containers/proxy-httpd/uyuni-configure.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+
+import os
+import yaml
+
+config_path = "/etc/uyuni/"
+
+# read from file
+with open(config_path + "config.yaml") as source:
+    config = yaml.safe_load(source)
+    
+    # Save the systemid file
+    with open("/etc/sysconfig/rhn/systemid", "w") as file:
+        file.write(str(config['system_id']))
+    
+    # Decode and install SSL CA certificate
+    os.system(f"echo -n '{config['binary_ca_certs']}' | base64 -d > /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT")
```

Comment:
I'd recommend base64 decoding and file writing to happen in Python, the intention being to shell out (eg. call `system()`) only when necessary.